src/test/java/io/kestra/plugin/weaviate/QueryTest.java

```diff
@@ -0,0 +1,138 @@
+package io.kestra.plugin.weaviate;
+
+import io.kestra.core.runners.RunContext;
+import io.kestra.core.runners.RunContextFactory;
+import io.kestra.core.serializers.JacksonMapper;
+import io.kestra.core.storages.StorageInterface;
+import io.micronaut.test.extensions.junit5.annotation.MicronautTest;
+import jakarta.inject.Inject;
+import org.apache.commons.io.Charsets;
+import org.apache.commons.io.IOUtils;
+import org.hamcrest.Matchers;
+import org.junit.jupiter.api.Test;
+
+import java.nio.charset.Charset;
+import java.util.List;
+import java.util.Map;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.*;
+
+@MicronautTest
+public class QueryTest {
+
+    public static final String SCHEME = "http";
+    public static final String HOST = "localhost:8080";
+    private static final String QUERY = """
+                       {
+  Get {
+    QueryTest (
+      limit: 50
+    ) {
+      title
+    }
+  }
+}
+                       """;
+
+    @Inject
+    private RunContextFactory runContextFactory;
+
+    @Inject
+    private StorageInterface storageInterface;
+
+    @Test
+    public void testQuery_Without_Internal_Storage() throws Exception {
+        RunContext runContext = runContextFactory.of();
+
+        String className = "QueryTest";
+        List<Map<String, Object>> parameters = List.of(Map.of("title", "test success"));
+
+        BatchCreate.Output batchOutput = BatchCreate.builder()
+            .scheme(SCHEME)
+            .host(HOST)
+            .className(className)
+            .parameters(parameters)
```

Comment:
With remarks from above, parameters (which is going to be objects) can now be a Kestra Internal Storage URI.
See [this](https://github.com/kestra-io/plugin-aws/commit/c2b188c3c2fdaff2b56630d2d22176c7fdc517ec#diff-3f16fb669443375f8087cfde129c851dbe0aa4a885a933b2245b4373e2a7dfb1R154-R172) to make a test on it